apps/watch/src/components/PageCollections/CollectionVideoContentCarousel/SeeAllButton/SeeAllButton.tsx

```
@@ -0,0 +1,37 @@
+   */
+  text: string
+  /**
+   * The function to call when the button is clicked
+   */
+}
+
+export const SeeAllButton = ({ text }: SeeAllButtonProps): ReactElement => {
+  return (
+    <a
+      href="https://www.jesusfilm.org/watch?utm_source=jesusfilm-watch"
+      target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)